src/Time/Tags/TimeAndPrevious.hpp

```diff
@@ -17,9 +17,9 @@
 namespace Tags {
 struct Time;
 }  // namespace Tags
-namespace evolution::Tags {
+namespace Tags {
```

Comment:
[optional] Merge the duplicate `Tags` forward-declaration namespaces.  They occur in other files as well.